shared/common-adapters/input.native.js

```diff
@@ -13,38 +13,24 @@ import type {Props} from './input'
 type State = {
   focused: boolean,
   height: ?number,
-  value: string,
```

Comment:
no need to copy `value` from props to state

Reply:
i don't understand this change. we manage the local text state in value right. so if we remove this how can this be used w/o being driven by a prop?

Reply:
Per our discussion, put this back into state for uncontrolled components. Also tested with an uncontrolled component (password field on login).